.gitignore

```diff
@@ -6,14 +6,12 @@
 npm-debug.log
 npm-debug.log.*
 *.dat
-package-lock.json
```

Comment:
I would not commit package-lock.json, rest ok.

Reply:
How do you make sure you get repeatable builds without package-lock?

Reply:
With tests.
Just my opinion and personal advice: _I would not commit package-lock.json on modules (only if the module is "big enough" and there is a bigger amount of collaborators), but only on projects._